corehq/apps/reports/static/reports/js/filters/case_list_explorer_knockout_bindings.js

```diff
@@ -63,12 +67,21 @@ hqDefine("reports/js/filters/case_list_explorer_knockout_bindings", ['jquery', '
                         return;
                     }
                     callback(null, _.map(caseProperties, function (suggestion) {
+                        var currentLabelLength = suggestion.name.length,
+                            metaText = suggestion.case_type || suggestion.meta_type,
```

Comment:
is it guaranteed that at least one of `case_type` or `meta_type` will exist?

Reply:
I believe so